src/pkg/services/m365/api/users.go

```diff
@@ -295,6 +303,27 @@ func (c Users) GetInfo(ctx context.Context, userID string) (*UserInfo, error) {
 
 	userInfo.Mailbox = mbxInfo
 
+	if mfs != nil {
+		mf := mfs.GetValue()[0] // we will always have one
+		options := &users.ItemMailFoldersItemMessagesDeltaRequestBuilderGetRequestConfiguration{
+			QueryParameters: &users.ItemMailFoldersItemMessagesDeltaRequestBuilderGetQueryParameters{
+				Top: ptr.To[int32](1), // just one item is enough
+			},
+		}
+		_, err = c.stable.Client().
```

Comment:
Can you please add a todo that this is a quick hack to interpret over-quota errors, and that future changes could instead look up the mailbox quota status itself to evaluate delta usage?